src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -96,6 +98,20 @@ private CallbackResponse handleDefendAllClaim(CallbackParams callbackParams) {
         }
         if ("DEFENDANT_RESPONSE_SPEC".equals(callbackParams.getRequest().getEventId())) {
 
+            if ("HAS_PAID_THE_AMOUNT_CLAIMED".equals(caseData.getDefenceRouteRequired())
```

Comment:
Can we create an Interface for constants please. Its much easier to manage it that way.

Reply:
this is not sonarqube compliant
https://rules.sonarsource.com/java/RSPEC-1214

so did this

public final class Status {             // Compliant
   public static final int OPEN = 1;
   public static final int CLOSED = 2;
}